server/.vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Is this file created by default while setting up the boilerplate .NET API? If not, can you please remove the `.vscode` folder and add `.vscode` to the gitignore?

Reply:
Yep thats my bad, I've added it now